afs/afs-ext-base/src/main/java/com/powsybl/afs/ext/base/ImportedCaseBuilder.java

```diff
@@ -150,8 +150,12 @@ public ImportedCase build() {
             throw new UncheckedIOException(e);
         }
         context.getStorage().setConsistent(info.getId());
+
         context.getStorage().flush();
 
+        context.getStorage().getEventStore().pushEvent(new CaseImported(info.getId(),
```

Comment:
To really distinguish event store and storage it would be better not getting event store from storage and pass it through ProjectFileBuildContext instead.

Reply:
I'm not ok with that.  we should be able of getting an eventsStore from a storage (see AbstractAppStorageTest to add a listener) otherwise we have to create an define an eventsStore instance each time we would use it later.